client_source/sqlplus/ut_run.sql

```diff
@@ -70,7 +85,12 @@ Parameters:
 
 whenever sqlerror exit failure
 whenever oserror exit failure
-conn &1
+
+define client_path="&1"
```

Comment:
We could remove only the invocation examples from ut_run.sql, as they are already in readme.